src/components/App.js

```diff
@@ -84,7 +84,15 @@ export default class App extends React.Component {
       <BrowserRouter>
         <Layout>
           <QueryActions />
-          <QueryServiceInfo />
+
+          {/* XXX Not sure what point there is to ping Normandy Recipe-server
+          unless you're logged in here. */}
+          {this.props.accessToken ? (
+            <QueryServiceInfo
```

Comment:
I find it quite weird that opening the delivery console, automatically triggers a check to Normandy. I haven't even indicated that I'm interested in doing Normandy stuff.